### ISSUE TITLE
Determine the logic behind the Vectronic data ID

### ISSUE THREAD

**Issue by HansMartinMT** [Huh/collar#24]
We want to determine if we can use the data point ID to allow us to access data starting after a previous download. If the data point ID's are unique to 'our' collars, this could allow us to use a single ID to access for all of the collars instead of having a unique data point ID for each collar.

**Comment by rgzn** [comment]
By data point ID, do you mean the first column in a vectronic text file?  Eg. the "2674" in the following snippet:

`    2674    16245 2019-01-01 20:00:31 2019-01-01 12:00:31`

Just curious if I can help answer your question. I have lots of these to look at. At first glance, they appear to just increment with each position record for a collar. Not unique to collars or accounts.

**Comment by Huh** [comment]
Yeah, one guess is that they increment across some large set of collars or
even all active collars?

**Comment by Huh** [comment]
@HansMartinMT or @foresthayes can you please respond to this question "By data point ID, do you mean the first column in a vectronic text file?"

**Comment by foresthayes** [comment]
Using collar::fetch_vectronics the first column I get back is "idPosition". Below are the last 50 values **for a single collar**.  I have not been able to figure out what these values relate to although they do seem to always increase.  Note that the increment is often ~50,000 but is also highly variable.

```
  [1] 68532723 68583097 68674209 68721084 68771585 68822193 68870519 68919401
  [9] 68969061 69014782 69061304 69201464 69394297 69628966 69675247 69721153
 [17] 69765153 69809496 69902881 69955247 70003498 70053520 70101822 70153965
 [25] 70202041 70252035 70303758 70354692 70744163 70865104 70922193 71037156
 [33] 71096907 71151995 71209254 71262657 71317654 71371761 71479678 71482467
 [41] 71519533 71597320 71651912 71710344 71774644 71834846 71918915 72014790
 [49] 72120079 72127339 
```
